### PR TITLE
Fortinet FortiManager Device Add Module

### PR DESCRIPTION
**Pull request by ftntcorecse**
##### SUMMARY

This module adds/discovers devices to a Fortinet FortiManager via API/Ansible.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

This module adds/discovers devices to a Fortinet FortiManager via API. 

##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0 (devel 4b121ea2ad) last updated 2018/09/20 13:34:55 (GMT -700)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/usr/ansible_ftnt/library']
  ansible python module location = /root/ansibleSource/ansible/lib/ansible
  executable location = /root/ansibleSource/ansible/bin/ansible
  python version = 2.7.12 (default, Nov 20 2017, 18:23:56) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ftntcorecse** [comment]
Any word on this guys? We've got more coming this week.

File: lib/ansible/modules/network/fortimanager/fmgr_device.py [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+options:
+  adom:
+    description:
+      - The ADOM the configuration should belong to.
+    required: true
+    default: root
+  host:
+    description:
+      - The FortiManager's Address.
```
Comment (samccann) [inline lib/ansible/modules/network/fortimanager/fmgr_device.py]:
```suggestion
      - The FortiManager's address.
```

File: lib/ansible/modules/network/fortimanager/fmgr_device.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    description:
+      - The username of the device being added to FortiManager.
+    required: false
+  device_password:
+    description:
+      - The password of the device being added to FortiManager.
+    required: false
+  device_ip:
+    description:
+      - The IP of the device being added to FortiManager.
```
Comment (samccann) [inline lib/ansible/modules/network/fortimanager/fmgr_device.py]: Can this support both IPv4 and IPv6 addresses?

Comment (ftntcorecse) [reply lib/ansible/modules/network/fortimanager/fmgr_device.py]: Yes. Made a change to reflect this.

**Comment by ftntcorecse** [comment]
@samccann Made changes to grammar and fixed pylint issue with unneeded-not. Thanks.

**Review by samccann — APPROVED** [review]
looks good from a docs perspective.

**Review by NilashishC — APPROVED** [review]

**Comment by NilashishC** [comment]
LGTM

**Pull request merged**